styleguide-app/Examples/RadioButton.elm

```diff
@@ -102,6 +110,62 @@ viewVanilla state =
             , selectedValue = state.selectedValue
             , onSelect = Select
             , valueToString = identity
+            , disclosure = Nothing
+            , describedBy = []
+            }
+        ]
+
+
+viewDisclosures : State -> Html Msg
+viewDisclosures state =
+    div []
+        [ RadioButton.view
+            { label = "Reset to default class password"
+            , value = "DefaultPassword"
+            , name = "radio-button-examples"
+            , selectedValue = state.selectedValue
+            , onSelect = Select
+            , valueToString = identity
+            , describedBy = [ "default-class-password" ]
```

Comment:
I'm not sure that, as a developer using the RadioButton, I would understand what values to pass in to `describedBy` 🤔

For instance, why do I/how would I know to pass in the id for the "The default class password is" element but not the id for the "Copy password" button?